### DISCUSSION TITLE
Unicode U+0019 being rejected by Lexer

### DISCUSSION THREAD

**Discussion by tao-zhang-24** [webonyx/graphql-php#1448] [Q&A]
Hi all,

Not too sure whether this is the right place to discuss this issue that has been bothering me for a while.

I always have a problem with sending a query that contains "invalid" unicode U+0019 , or \u0019 as the query is generated by a python service, e.g. `address: "Main\u00a0South\u00a0Rd,\u00a0O\u0019 Halloran`. Please correct me if I was wrong but I believe that this 'End of Medium' is considered as a valid unicode under UTF-8 standard. Shouldn't the GraphQL parser be accepcting it instead of throwing a `SyntaxError` from `Lexer.php`? Or is there a way to walk-around this check as we really would like to just skip this mutation/query validation step.

Any helps or information will be much appreciated.

Framework: Symfony API Platform v2.7.0
graphqp PHP version: v14.11.6
PHP version: v8.0

**Comment by spawnia** [comment]
Rightfully so, see https://spec.graphql.org/October2021/#SourceCharacter.
